Library/OptimizelyFeature.swift

```diff
@@ -3,13 +3,15 @@ import Foundation
 public enum OptimizelyFeature: String, CaseIterable {
   case commentFlaggingEnabled = "ios_comment_threading_comment_flagging"
   case navigationSelectorProjectPageEnabled = "project_page_v2"
+  case projectPageStoryTabEnabled = "project_page_v2_story"
```

Comment:
wonder if we should prefix this with ios or native? is android using the same flag?

Reply:
android hasn't added the flag on optimizely yet. I see they are prefixing theres' with `android`, but the ticket said `project_page_v2_story` which is what I went with, happy to change it later if that's cool.